### ISSUE TITLE
Bundle multiple modules together

### ISSUE THREAD

**Issue by j-f1** [Rich-Harris/packd#64]
I’ve been using bundle.run for some [Observable](https://ObservableHQ.com) notebooks, but I sometimes need to import multiple modules that depend on each other. Would it be possible to add a way to import multiple modules as an object?

**Comment by xtuc** [comment]
Yes, that would be great! We could pass the packages seperated by a commas in the URL, it will bundle them and return an object.

I'm wondering if genreating a file that requires the packages would work, something like:
```js
const packages = {
a: require("a"),
b: require("b")
};
```

I'm missing time to implement such a feature, would you mean interested in helping here?
